### PR TITLE
add doc for reserved metadata

### PR DESCRIPTION
**Pull request by superyyrrzz**
add doc for gitContribute #505
 @chenkennt @ansyral @vwxyzh @qinezh @hellosnow

### CODE REVIEW

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 1]
```diff
@@ -277,6 +277,23 @@ Please refer to [How to Create Custom Templates](howto_create_custom_template.md
...
+----------------------|---------|---------------------------
+_appTitle             | string  | Will be appended to each output page's head title.
+_appFooter            | string  | The footer text. Will show DocFX's Copyright text if not specified.
+_enableSearch         | bool    | Indicate whether show the search box on the top of page.
+_disableNavbar        | bool    | Indicate whether to show the navigation bar on the top of page.
+_disableBreadcrumb    | bool    | Indicate whether to show breadcrumb on the top of page.
+_disableToc           | bool    | Indicate whether to show table of contents on the left of page.
+_disableAffix         | bool    | Indicate whether to show the affix bar on the right of page.
+_disableContribution  | bool    | Indicate whether to show the `View Source` and `Improve this Doc` buttons.
+gitContribute         | object  | Customized the `Improve this Doc` URL button for public contributors. Use `repo` to specify the contribution repository URL. Use `branch` to specify the contribution branch. Use `path` to specify the folder for new overwrite files .
```
Comment (chenkennt) [inline Documentation/tutorial/docfx.exe_user_manual.md]: shall we use _gitContribute to make this metadata consistent with others?

Comment (chenkennt) [inline Documentation/tutorial/docfx.exe_user_manual.md]: Customized -> Customize

Comment (superyyrrzz) [inline Documentation/tutorial/docfx.exe_user_manual.md]: Yes. doc issued is fixed, will rename it in code in another PR.

**Comment by chenkennt** [comment]
:+1:

**Comment by qinezh** [comment]
:shipit:

File: Documentation/tutorial/docfx.exe_user_manual.md (hunk 1 above)
Comment (hellosnow) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >gitContribute [](start = 0, length = 13)

how about give an example for gitContribute here?

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 2]
```diff
@@ -277,6 +277,23 @@ Please refer to [How to Create Custom Templates](howto_create_custom_template.md
...
 
+#### 3.2.2 Reserved Metadata
+After passing values through global metadata or file metadata, DocFX can use these metadata in templates to control the output html.
+Reserved metadatas:
+
+Metadata Name         | Type    | Description
+----------------------|---------|---------------------------
+_appTitle             | string  | Will be appended to each output page's head title.
+_appFooter            | string  | The footer text. Will show DocFX's Copyright text if not specified.
+_enableSearch         | bool    | Indicate whether show the search box on the top of page.
```
Comment (hellosnow) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >whether show [](start = 43, length = 12)

whether to show?

**Comment by hellosnow** [comment]
:shipit:

**Comment by vwxyzh** [comment]
:shipit:

File: Documentation/tutorial/docfx.exe_user_manual.md (hunk 2 above)
Comment (superyyrrzz) [inline Documentation/tutorial/docfx.exe_user_manual.md]: resolved

**Pull request merged**